conf/test_illumina.config

```
@@ -23,4 +23,6 @@ params {
     input           = 'https://raw.githubusercontent.com/Arcadia-Science/test-datasets/main/metagenomics/illumina/samplesheet_test.csv'
     platform        = 'illumina'
     sourmash_dbs    = 'https://raw.githubusercontent.com/Arcadia-Science/test-datasets/main/metagenomics/sourmash_dbs/sourmash_dbs_test.csv'
+    diamond_db      = 'https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/diamond_dbs/test-cheese-proteins-db.dmnd'
+    diamond_columns = 'qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore corrected_bitscore qlen slen'
```

```suggestion
    diamond_columns = 'qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore corrected_bitscore qlen slen qcovs qcovhsp'
```

realized two more that would be useful!